patcher/Patcher.py

```
@@ -489,7 +465,7 @@ def replace_data(new_data, offset):
     if SECT_SIZE > 0:
         if SECT_SIZE < exxt_sect.f_size:
             raise Exception(
-                f"Section size too small. Required: 0x{exxt_sect.f_size:x}")
+                f"Section size too small. Required: 0x{exxt_sect.f_size: x}")
```

Ruff (0.14.14)

Create your own exception

(TRY002)